docs/00-introduction.ipynb

```diff
@@ -71,7 +71,8 @@
     "    utterances=[\n",
     "        \"isn't politics the best thing ever\",\n",
     "        \"why don't you tell me about your political opinions\",\n",
-    "        \"don't you just love the president\" \"don't you just hate the president\",\n",
+    "        \"don't you just love the president\",\n",
+             \"don't you just hate the president\",\n",
```

Comment:
At the moment this is invalid notebook code because it is missing a `"` — you can fix by changing it to:

```
    "        \"don't you just love the president\",\n",
    "        \"don't you just hate the president\",\n",
```

Reply:
Oh right! Thanks it's fixed :)